lib/oref0-setup/mdt-cgm.json

```diff
@@ -125,15 +125,15 @@
   },
   {
      "get-bg": {
-      "command": "! bash -c \" (echo -n MDT cgm data retrieve && openaps monitor-cgm 2>/dev/null >/dev/null && grep -q glucose cgm/cgm-glucose.json && echo d) && cp -pu cgm/cgm-glucose.json cgm/glucose.json && cp -pu cgm/glucose.json monitor/glucose-unzoned.json && (echo -n MDT cgm data reformat && openaps report invoke monitor/glucose.json 2>/dev/null >/dev/null && echo ted)\""
+      "command": "! bash -c \" (echo -n MDT cgm data retrieve && openaps monitor-cgm 2>/dev/null >/dev/null && grep -q glucose cgm/cgm-glucose.json && echo d) && cp -pu cgm/cgm-glucose.json cgm/glucose.json && cp -pu cgm/glucose.json monitor/glucose-unzoned.json && (echo -n MDT cgm data reformat && openaps report invoke monitor/glucose.json nightscout/glucose.json 2>/dev/null >/dev/null && echo ted)\""
      },
     "type": "alias",
     "name": "get-bg"
   },
   {
     "type": "alias",
     "upload-bg": {
-      "command": "report invoke nightscout/glucose.json nightscout/recent-missing-entries.json nightscout/uploaded-entries.json"
+      "command": "report invoke nightscout/recent-missing-entries.json nightscout/uploaded-entries.json"
```

Comment:
I agree all the conversion of the enlite data is confusing. but since the enlite is the only CGM that uploads to nightscout, we have to have the correct format. And it took some very smart people a long time to get that all to work. You are more than welcome to try to simplify it :-) 

to answer your question
the reason to use the nightscout/glucose.json because that is the only cgm data that have the date field that is being used by jq